docs/source/cli/create_connector.rst

```diff
@@ -0,0 +1,205 @@
+.. _cli.create_connector:
+
+===============
+Create Connector
+===============
+
+Description
+~~~~~~~~~~~
+
+The create connector command allows users to create a connector, including configuring IAM roles, mapping backend roles, and creating secrets automatically. Users can create a connector either interactively or by using a configuration file. Connectors can be created by simply selecting from a list of supported connectors and their associated models. The CLI then guides them through providing only the essential model-specific information, such as AWS region for Amazon services.
```

Comment:
> including configuring IAM roles, mapping backend roles, and creating secrets automatically.

This part looks confusing. This one is only for AWS OpenSearch Service. Let's clarify.